pkgs/development/python-modules/anova-wifi/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, aiohttp
+, sensor-state-data
+, pytestCheckHook
+, pytest-asyncio
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "anova-wifi";
+  version = "0.10.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "Lash-L";
+    repo = lib.replaceStrings ["-"] ["_"] pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-tCmvp29KSCkc+g0w0odcB7vGjtDx6evac7XsHEF0syM=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    sensor-state-data
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-asyncio
+    pytest-cov
```

Comment:
Patch out `--cov` arguments here https://github.com/Lash-L/anova_wifi/blob/main/pyproject.toml#L49.

We don't care one bit about test coverage.